.github/workflows/ci_build.yml

```
@@ -0,0 +1,40 @@
+# This workflow will build a package using Maven and then publish it to GitHub packages when a release is created
+# For more information see: https://github.com/actions/setup-java#apache-maven-with-a-settings-path
+
+name: CI
+
+on:
+  push:
+    branches: [ master ]
```

What about other supported versions? I believe this should be:
```suggestion
    branches:
    	- master
    	- ver/*
```
The same with PRs.